hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -2590,7 +2620,20 @@ String getOperationType() {
 
     @Override
     String getOperationType() {
-      return "ENABLE";
+      return "MODIFY";
+    }
+  }
+
+  private static class ModifyTableStoreFileTrackerProcedureBiConsumer
+    extends TableProcedureBiConsumer {
+
+    ModifyTableStoreFileTrackerProcedureBiConsumer(AsyncAdmin admin, TableName tableName) {
+      super(tableName);
+    }
+
+    @Override
+    String getOperationType() {
+      return "MODIFY_STORE_FILE_TRACKER";
```

Comment:
Because we use MODIFY instead of MODIFY_TABLE here, so I assume we just name it MODIFY_STORE_FILE_TRACKER is enough. I'm fine with changing it to MODIFY_TABLE_STORE_FILE_TRACKER